tests/integration/test_cloudformation.py

```diff
@@ -73,10 +74,6 @@
 TEST_TEMPLATE_9 = (
     """
 Parameters:
-  FeatureBranch:
```

Comment:
FYI: not sure what the purpose of these was but considering the "FeatureBranch" is actually the name of the Condition in the template, the Parameter doesn't make much sense here

Reply:
good catch! 👍